projects/1.18/assets/create-cafe/createcafe/lang/zh_cn.json

```
@@ -1 +1,171 @@
-{}
\ No newline at end of file
+{
+    "block.createcafe.melted_sugar": "融化的糖",
+    "block.createcafe.filling": "填充",
+  
+    "tooltip.createcafe.res": "防火 (0:12)",
+    "tooltip.createcafe.regen": "重生 (0:13)",
+    "tooltip.createcafe.strength": "力量 (0:10)",
+    "tooltip.createcafe.speed": "速度 (0:13)",
+    "tooltip.createcafe.super": "防火 (0:13), 速度 (0:13), 力量 (0:13), 重生 (0:13)",
+  
+    "itemGroup.createCafeTab": "创意咖啡馆模组",

```

```suggestion
    "itemGroup.createCafeTab": "机械动力：奶茶店",
```
嘛我知道cafe一般是`咖啡馆`，不过cafe广义是指“街边贩售饮料和甜点的小店”，显然这个mod里90%以上的饮品都是奶茶而非咖啡。
创意咖啡馆就更离谱了。

---

塘埔冰室